brain/python/helpers.h

```diff
@@ -57,6 +59,15 @@ inline boost::python::list toPythonList( const std::vector< T >& vector )
     return result;
 }
 
+inline uint32_ts toVector( const GIDSet& gids )
```

Comment:
See lines 85-101 in brain/python/circuit.cpp, without the helper function you need a temporary variable.